cloudferrylib/os/actions/transport_instance.py

```diff
@@ -18,6 +18,7 @@
 from cloudferrylib.base.action import action
 from cloudferrylib.os.identity import keystone
 from cloudferrylib.utils import utils as utl
+from cloudferrylib.os.compute import nova_compute
```

Comment:
move to position between `from cloudferrylib.base.action import action` and `from cloudferrylib.os.identity import keystone`. Try to comply with an openstack importing standard: http://docs.openstack.org/developer/hacking/#imports